### PR TITLE
3.2/4.0 version issues

### PR DESCRIPTION
**Pull request by keichwa**
@paususe here are my findings

### CODE REVIEW

File: modules/retail/pages/retail-install-setup.adoc [hunk 1]
```diff
@@ -94,22 +95,28 @@ For instructions to install Salt-based proxy minions and register them to {susem
...
+////
+sle-product-sles15-sp1-updates-x86_64
+sle-product-suse-manager-retail-branch-server-4.0-updates-x86_64
+sle-module-server-applications15-sp1-updates-x86_64-smrbs-4.0
+sle-module-basesystem15-sp1-updates-x86_64-smrbs-4.0`
+
+sle-manager-tools15-updates-x86_64-sp1
+sle-module-desktop-applications15-sp1-updates-x86_64
+and respective pools
+////
```
Comment (aaannz) [inline modules/retail/pages/retail-install-setup.adoc]: I was too fast to copy from testsuite.

* sle-product-suse-manager-retail-branch-server-4.0-updates-x86_64
* sle-module-server-applications15-sp1-updates-x86_64-smrbs-4.0
* sle-module-basesystem15-sp1-updates-x86_64-smrbs-4.0

and respective pools are needed for Branch Server proxy.

The rest is for generic SLE15 systems, not sure how we got desktop applications in the mix.

**Comment by keichwa** [comment]
Ondřej Holeček <notifications@github.com> writes:

>
> * sle-product-suse-manager-retail-branch-server-4.0-updates-x86_64
> * sle-module-server-applications15-sp1-updates-x86_64-smrbs-4.0
> * sle-module-basesystem15-sp1-updates-x86_64-smrbs-4.0
>
> and respective pools are needed for Branch Server proxy.

Thanks for checking!

-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Mary Higgins, Sri Rasiah, HRB 21284 (AG Nürnberg)

**Comment by keichwa** [comment]
Pau Garcia Quiles <notifications@github.com> writes:

> paususe approved this pull request.
>
> I don't know if these are all the required changes but they definitely
> look good

Thanks for reviewing!  When approved by co-writers I'll merge it.  Then
we can read it on opensource.suse.com, and actually see what's in.  And
do some quick fixes if needed.

-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Mary Higgins, Sri Rasiah, HRB 21284 (AG Nürnberg)

**Comment by keichwa** [comment]
> ```
> // FIXME: Update this URL when we are about to the 4.0 retail documentation
> // ke, 2019-06-19
> ```
> 
> This link is now available on suse.com docs :) Otherwise looks good!

Yes, in the meantime, I deleted the comment.  We now have an (internal) xref in this file.

**Comment by keichwa** [comment]
merging now as proposed by joseph :)

**Pull request merged**